MessengerOnFileForLinux/FileHandling/src/FileHandling.cpp

```diff
@@ -282,15 +282,26 @@ bool FileInterface::Managment::removeFile(const std::string& pathToFile)
 
 bool FileInterface::Modification::removeRow(const std::string& pathToFile, const std::string& pattern)
 {
+    std::string logInfo = "FileInterface::Modification::removeRow() -> " + pathToFile;
+    fileLog(logInfo.c_str(), LogSpace::FileHandling);
+
+    if (nullptr == Accesor::getRow(pathToFile, pattern))
+    {
+        const std::string logInfo = "FileInterface::Modification::removeRow() -> Attempt to remove unexist row!";
```

Comment:
static? Czy w sumie zbędne? Ewentualnie jakoś to wybić??

Reply:
?